config/cert-manager/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+bases:
+- https://github.com/cert-manager/cert-manager/releases/download/v1.11.0/cert-manager.yaml
```

Comment:
We don't need this one anymore. It is now being installed as a pre-requisite by `make deploy`